test/versioned/fastify/fastify-2-naming.tap.js

```diff
@@ -63,16 +64,24 @@ let testCount = 0
 const testUri = (uri, agent, test, port) => {
   agent.on('transactionFinished', (transaction) => {
     testCount++
-    test.equals(
+    test.equal(
```

Comment:
Do we have any tests that verify we are not going to end up with any raw route parameters or query string items in the naming?

I'm assuming this works but not sure if we have any coverage. These seem to just be hitting simple defined routes VS say '/person/111222333/' and ensuring we end up with '/person/:id' as the name (or whatever matches fastify-style).

If we don't have any coverage, perhaps should have a story for that.

Reply:
we don't but i'll add to this https://github.com/newrelic/node-newrelic/issues/928 which was to flesh out our integration tests for fastify